dip-0003.md

```diff
@@ -232,8 +267,15 @@ Reorganisations of the chain must also correctly undo changes to the chain-tips
 
 A ProRegTx is invalid if any of these conditions are true:
 
+In the case of type 0:
   1. collateralOutpoint `hash` is null but an output with 1000 DASH is not present at position `n` of the ProRegTx outputs
   2. collateralOutpoint `hash` is not null but an output with 1000 DASH can't be found in the UTXO specified by the `hash` and `n`
+
+In the case of type 1:
+  1. collateralOutpoint `hash` is null but an output with 4000 DASH is not present at position `n` of the ProRegTx outputs
+  2. collateralOutpoint `hash` is not null but an output with 4000 DASH can't be found in the UTXO specified by the `hash` and `n`
+
+Applicable to any type:
```

Comment:
This one still isn't working if you look at the preview. Try adding a blank line / restarting numbering at 1.
```suggestion
Applicable to any type:

```

Reply:
@thephez Should be good now